fitlins/interfaces/afni.py

```diff
@@ -0,0 +1,637 @@
+# coding: utf-8
+from pathlib import Path
+import os
+import os.path as op
+import numpy as np
+import pandas as pd
+import subprocess as sp
+import io
+from warnings import warn
+
+
+from nipype.interfaces.base import isdefined
+import nibabel as nb
+
+from .nistats import FirstLevelModel, prepare_contrasts, _flatten
+from .utils import MergeAll
+
+STAT_CODES = nb.volumeutils.Recoder(
+    (
+        (0, "none", "none"),
+        (1, "none", "none"),
+        (2, "correlation", "Correl"),
+        (3, "t test", "Ttest"),
+        (4, "f test", "Ftest"),
+        (5, "z score", "Zscore"),
+        (6, "chi2", "Chisq"),
+        (7, "beta", "Beta"),
+        (8, "binomial", "Binom"),
+        (9, "gamma", "Gamma"),
+        (10, "poisson", "Poisson"),
+        (11, "normal", "Normal"),
+        (12, "non central f test", "Ftest_nonc"),
+        (13, "non central chi2", "Chisq_nonc"),
+        (14, "logistic", "Logistic"),
+        (15, "laplace", "Laplace"),
+        (16, "uniform", "Uniform"),
+        (17, "non central t test", "Ttest_nonc"),
+        (18, "weibull", "Weibull"),
+        (19, "chi", "Chi"),
+        (20, "inverse gaussian", "Invgauss"),
+        (21, "extreme value 1", "Extval"),
+        (22, "p value", "Pval"),
+        (23, "log p value", "LogPval"),
+        (24, "log10 p value", "Log10Pval"),
+    ),
+    fields=("code", "label", "stat_code"),
+)
```

Comment:
Pretty sure this is just `nibabel.nifti1.intent_codes`, restricted to the first 24?

Reply:
The intent codes in nifti mirror the afni brik stat codes (they were copied from them). I don't know what's more intuitive to describe it as a stat code mapping or to add the brik labels as aliases to the niftit intent code mapping. I think moving this to nibabel would be good though.

Reply:
Since they're in NIFTI, I would probably just stick with that, as it's all official and whatnot. But if you wanted to add them to `nibabel.brikhead`, you're welcome to.